oio/api/backblaze.py

```diff
@@ -86,15 +86,20 @@ def _upload_chunks(self, conn, size, sha1, md5, temp):
         while True:
             self.meta_chunk['size'] = size
             try:
-                logger.debug("sha1: %s", sha1)
                 conn['backblaze'].upload(self.backblaze_info['bucket_name'],
                                          self.sysmeta, temp, sha1)
                 break
             except BackblazeException as b2e:
                 temp.seek(0)
                 if try_number == 0:
+                    logger.debug('headers sent : %s'
+                                 % str(b2e.headers_send))
                     raise OioException('backblaze upload error: %s'
                                        % str(b2e))
+                else:
+                    val_tmp = pow(2, TRY_REQUEST_NUMBER - try_number)
+                    sleep = b2e.headers_received.get("Retry-After", val_tmp)
+                    time.sleep(sleep)
```

Comment:
please don't use time.sleep in all this API code, this will block everything
you should instead use `eventlet.sleep()`